fhir-model/src/main/java/com/ibm/fhir/model/util/ValidationSupport.java

```diff
@@ -370,6 +374,35 @@ private static SchemaFactory createSchemaFactory() {
         return elements;
     }
 
+    /**
+     * @return the same list that was passed
+     * @throws IllegalStateException if the passed list contains any null objects or objects of an incompatible type
```

Comment:
also throws IllegalStateException if passed list is empty

Reply:
updated the javadoc to cover this